site/content/learn/headless/basics-emulating-mobile-devices.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Emulating mobile devices
+subTitle: Learn how to emulate mobile devices
+date: 2022-07-19
+author:
+  - Giovanni Rago
+  - Stefan Judis
+githubUser:
+  - ragog
+  - stefanjudis
+tags:
+  - basics
+
+weight: 8
+
+menu:
+  learn:
+    parent: "Getting started"
+---
+
+Puppeteer and Playwright control headless desktop browsers but can also emulate mobile devices. And while mobile device emulation can't replace testing on mobile devices, it's a practical and quick-to-setup approach to test your site in a mobile scenario.
```

Comment:
@stefanjudis do you think we could spell out why it still makes sense to use this approach, given how it doesn't replace testing on mobile? Can we claim it is still enough to test responsiveness? The question that pops in my mind after reading this intro is "ok it's easy to set up but is not the real thing... how close to real is it though? what _can_ I do with it?"

Reply:
Great feedback. let's improve!